docs/demos/manipulation.md

```diff
@@ -50,12 +50,16 @@ This demo showcases the capabilities of RAI (Robotec AI) in performing manipulat
    ros2 launch examples/manipulation-demo.launch.py game_launcher:=path/to/RAIManipulationDemo.GameLauncher
    ```
 
-2. In the second terminal, run the interactive prompt:
+2. In the second terminal, run the cli prompt or streamlit interface:
 
    ```shell
    python examples/manipulation-demo.py
    ```
 
+   ```shell
+   streamlit run examples/manipulation-demo-streamlit.py
+   ```
+
```

Comment:
Please add a few words of explanation for the `cli prompt` and `streamlit interface`.
e.g. That the cli version provides a simple script showing the rai api that could be used in other places whereas the streamlit is better for trying the demo on it's own

Reply:
Done.